content/influxdb/v1.8/about_the_project/releasenotes-changelog.md

```diff
@@ -9,6 +9,27 @@ menu:
 v2: /influxdb/v2.0/reference/release-notes/influxdb/
 ---
 
+## v1.8.5 [TBD]
+### Features
+
+- Add the ability to find which measurements or shards are contributing to disk size with the new [`influx_inspect report-disk`](/influxdb/v1.8/tools/influx_inspect/#report-disk) command. Useful for capacity planning and managing storage requirements.
+- Add support to [`influx_inspect export`](/influxdb/v1.8/tools/influx_inspect/#export) to write to standard out (`stdout`) by adding a hyphen after the [`-out`](/influxdb/v1.8/tools/influx_inspect/#--out-export_dir-or--out--) flag. Using this option writes to `stdout`, and sends status messages to standard error (`stderr`).
+- Update meta queries (for example, SHOW TAG VALUES, SHOW TAG KEYS, SHOW SERIES CARDINALITY, SHOW MEASUREMENT CARDINALITY, and SHOW MEASUREMENTS) to check the query context when possible to respect timeout values set in the [`query-timeout` configuration parameter](/influxdb/v1.8/administration/config/#query-timeout--0s). Note, meta queries will check the context less frequently than regular queries, which use iterators, because meta queries return data in batches.
+- Optimize shard lookups in groups containing only one shard. Thanks @StoneYunZhao!
+
+### Bug fixes
+
+-  Previously, successful writes were incorrectly incrementing the `WriteErr` statistics. Now, successful writes correctly increment the `writeOK` statistics.
+- Correct JSON marshalling error format.
+- Previously, a GROUP BY query with an offset that caused an interval to cross a daylight savings change inserted an extra output row off by one hour. Now, the correct GROUP BY interval start time is set before the time zone offset is calculated.
+- Previously, `cp.Mux.Serve()` closed all net.Listener instances silently on error, resulting in "connection refused" errors to the meta node. Now, InfluxDB correctly logs all Listener closure errors.
```

Comment:
I think this is an Enterprise related fix since there isn't a meta node as part of OSS.

Reply:
Thank you, @timhallinflux-removed! @davidby-influx to be sure, should we remove this from change log, or is there two parts to the issue?

Reply:
The code went into influxdb/tcp/mux.go, so not specific to Enterprise.

Reply:
After discussion with @timhallinflux, suggested wording for the TCP issue is something like this:

"improved error logging for TCP connection closures"

Reply:
thank you-updated!